app/src/shared/components/TextControl/index.jsx

```diff
@@ -45,7 +45,7 @@ const TextControl = ({
     const el = useRef(null)
     const ref = innerRef || el
     const reverted: Ref<boolean> = useRef(false)
-    const [value, setValue] = useState(valueProp || '')
+    const [value, setValue] = useState(valueProp == null ? '' : valueProp)
```

Comment:
There's a few other places where `setValue(valueProp)` is called, should it apply the same default logic in those places too?

Reply:
I'm pretty sure in other places `setValue` is called always with a string. Will double-check.